app/packages/core/src/components/Modal/Group/DynamicGroup/carousel/DynamicGroupsFlashlightWrapper.tsx

```diff
@@ -209,6 +216,10 @@ export const DynamicGroupsFlashlightWrapper = () => {
     });
   }, [deferred, flashlight, updateItem, options, selected]);
 
+  useEffect(() => {
+    init();
+  }, [init]);
```

Comment:
making sure this is intentional

Reply:
Yes, it's a small detail to avoid redundant flashlight calls